apps/src/netsim/NetSimSendPanel.js

```diff
@@ -330,16 +345,19 @@ NetSimSendPanel.prototype.resetPackets_ = function () {
 NetSimSendPanel.prototype.onContentChange_ = function () {
   var nextBit = this.getNextBit_();
 
-  // Special case: If we have the "A/B" encoding enabled but _not_ "Binary",
-  // format this button label using the "A/B" convention
-  if (this.isEncodingEnabled_(EncodingType.A_AND_B) &&
-      !this.isEncodingEnabled_(EncodingType.BINARY)) {
-    nextBit = binaryToAB(nextBit);
-  }
+  if (nextBit === undefined) {
+    // If there are no bits queued up, disable the button
+    this.conditionallyToggleSetWireButton().text(i18n.setWire());
+  } else {
+    // Special case: If we have the "A/B" encoding enabled but _not_ "Binary",
+    // format this button label using the "A/B" convention
+    if (this.isEncodingEnabled_(EncodingType.A_AND_B) &&
+        !this.isEncodingEnabled_(EncodingType.BINARY)) {
+      nextBit = binaryToAB(nextBit);
+    }
 
-  this.getBody()
-      .find('#set-wire-button')
-      .text(i18n.setWireToValue({ value: nextBit }));
+    this.conditionallyToggleSetWireButton().text(i18n.setWireToValue({ value: nextBit }));
```

Comment:
This is super-picky (sorry): I feel like this is a misuse of chaining, specifically because I expect the `.text()` call to be applied to the first thing in the chain (`this`) or perhaps the most recent getter in the chain (there's not an obvious one here).  I sort of want to see either:

```javascript
this.getSetWireButton()
    .conditionallyToggleDisabled()
    .text(i18n.setWireToValue({ value: nextBit }));
```

or

```javascript
this.conditionallyToggleSetWireButton();
this.getSetWireButton().text(i18n.setWireToValue({ value: nextBit }));
```

Alternatively, `conditionallyToggleSetWireButton` is already calling `getNextBit_()` itself.  Is there any reason it shouldn't be renamed+expanded to also encapsulate setting the correct text for the button?